scripts/internal/manylinux-build-module-wheels.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
```

Comment:
I wonder if code with this script should be shared ? See https://github.com/InsightSoftwareConsortium/ITKPythonPackage/blob/master/scripts/internal/manylinux-build-wheels.sh